src/server/inversify.config.ts

```diff
@@ -82,20 +85,16 @@ export default () => {
   bindIfUnbound(DependencyIds.urlSearchService, UrlSearchService)
   bindIfUnbound(DependencyIds.deviceCheckService, DeviceCheckService)
 
-  const cloudmersiveScanService = new CloudmersiveScanService()
-
   container
     .bind(DependencyIds.allowedFileExtensions)
     .toConstantValue(DEFAULT_ALLOWED_FILE_EXTENSIONS)
   bindIfUnbound(DependencyIds.fileTypeFilterService, FileTypeFilterService)
-  container
-    .bind(DependencyIds.virusScanService)
-    .toConstantValue(cloudmersiveScanService)
+  bindIfUnbound(DependencyIds.virusScanService, CloudmersiveScanService)
   bindIfUnbound(DependencyIds.fileCheckController, FileCheckController)
 
-  container
-    .bind(DependencyIds.urlThreatScanService)
-    .toConstantValue(cloudmersiveScanService)
+  bindIfUnbound(DependencyIds.safeBrowsingMapper, SafeBrowsingMapper)
+  bindIfUnbound(DependencyIds.safeBrowsingRepository, SafeBrowsingRepository)
+  bindIfUnbound(DependencyIds.urlThreatScanService, SafeBrowsingService)
```

Comment:
Nitpick, but can we rename `urlThreatScanService` to `SafeBrowsingService` for consistency?

Reply:
Safe Browsing specifically refers to Google's service. The generic naming of the id is deliberate since we may want to replace or add services to vet the links